overviewer_core/world.py

```diff
@@ -834,18 +843,40 @@ def _get_block(self, palette_entry):
         elif key in ('minecraft:sunflower', 'minecraft:lilac', 'minecraft:tall_grass', 'minecraft:large_fern', 'minecraft:rose_bush', 'minecraft:peony'):
             if palette_entry['Properties']['half'] == 'upper':
                 data |= 0x08
-        elif key in ['minecraft:stone_slab', 'minecraft:sandstone_slab', 'minecraft:oak_slab',
-                     'minecraft:prismarine_slab','minecraft:dark_prismarine_slab','minecraft:prismarine_brick_slab',
-                     'minecraft:cobblestone_slab', 'minecraft:brick_slab',
-                     'minecraft:stone_brick_slab', 'minecraft:nether_brick_slab',
-                     'minecraft:quartz_slab', 'minecraft:petrified_oak_slab']:
+        elif key in wood_slabs + stone_slabs + prismarine_slabs:
+        # handle double slabs 
             if palette_entry['Properties']['type'] == 'top':
-                data += 8
+                data |= 0x08
             elif palette_entry['Properties']['type'] == 'double':
-                if 'oak' in key:
-                    block = 125
-                elif key == 'minecraft:stone_brick_slab':
-                    block = 98
+                if key in wood_slabs:
+                    block = 125 # block_double_wooden_slab
```

Comment:
at least two spaces before starting a comment on a line please, so
```python
block = 125  # block_double_wooden_slab
```

Also does this work fine for all types of wood?